torch/quantization/fx/quantize.py

```diff
@@ -926,6 +954,87 @@ def get_next_i(module, qparams):
             delattr(observed_root, n)
         return GraphModule(observed_root, self.quantized_graph)
 
+    def _fold_weight(self, quantized):
```

Comment:
can we add a high level docblock?  I think just adding what the PR has in the summary here would be helpful.